flake8_quotes/__init__.py

```diff
@@ -130,6 +132,27 @@ def get_quotes_errors(self, file_contents):
             }
 
 
+if not FLAKE8_VERSION2:
+    class QuoteChecker(QuoteCheckerBase):
```

Comment:
How about we define these as `Pep8QuoteChecker` and `NoPep8QuoteChecker`? Then, we can export based on `pep8` existing or not:

```python
QuoteChecker = Pep8QuoteChecker if pep8 else NoPep8QuoteChecker
```